addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -160,7 +161,9 @@ export const uploadService = {
                             file.uploaded = true;
                             await onUploaded(attachment);
                         }
-                        setTimeout(() => deleteFile(file.id), AUTOCLOSE_DELAY);
+                        // If error message length is over 60 characters, use AUTOCLOSE_DELAY_LONG to display the message for longer.
+                        let message_autoclose_delay = (file.hasError && file.errorMessage.length) < 60 ? AUTOCLOSE_DELAY : AUTOCLOSE_DELAY_LONG;
+                        setTimeout(() => deleteFile(file.id), message_autoclose_delay);
                     } catch (error) {
                         file.hasError = true;
                         setTimeout(() => deleteFile(file.id), AUTOCLOSE_DELAY);
```

Comment:
I don't think it's really useful to measure the message length, can't we just adjust `AUTOCLOSE_DELAY` to `8000` ?

Reply:
Yeah, I'll do that instead

(the other one applies if there's an error during the display process, so I figured there may not be an error message to measure)

Reply:
Note that this also displays non-error messages (e.g. "FIle successfully uploaded") for longer

Reply:
@creaturemusings In that case, we could have a delay for the success message (3000) and another delay if `file.hasError` (8000) :)